### PR TITLE
Issue #547 - Reformat JavaScript files

### PR DESCRIPTION
**Pull request by bigwave**
Simple 'Auto Format' in Visual Studio Code with the 'EditorConfig For VS Code' extension installed and the existing .editorconfig file.

Now you can have 'auto format on save/paste' configured and not have to worry about massive changes

### CODE REVIEW

File: javascripts/main.js [hunk 1]
```diff
@@ -1,75 +1,77 @@
-(function($) {
+(function ($) {
 
   var projectsSvc = new ProjectsService(projects),
-      compiledtemplateFn = null
-      projectsPanel = null;
+    compiledtemplateFn = null
+  projectsPanel = null;
```
Comment (shiftkey) [inline javascripts/main.js]: This seems off. Previous line probably needs a `,`

**Review by shiftkey — CHANGES_REQUESTED** [review]
Changes look good, aside from that one off line that's probably a syntax error...

**Comment by bigwave** [comment]
Fixed the syntax error...

**Review by shiftkey — APPROVED** [review]

**Comment by shiftkey** [comment]
@bigwave fantastic, thanks for that!

**Pull request merged**